### ISSUE TITLE
sql agent extension though installed does not appear on "manage:" home

### ISSUE THREAD

**Issue by scottkolle** [microsoft/azuredatastudio#16794]
Issue Type: <b>Bug</b>

sql agent extension though installed does not appear on "manage:" home
tried reinstallign azure data studio and uninstalling and reinstalling the agent extension.

Azure Data Studio version: azuredatastudio 1.31.1 (65fb22cc7c36db9c53af1ed2fdbdf48f66c682be, 2021-07-28T21:03:21.018Z)
OS version: Windows_NT x64 10.0.19043

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz (16 x 2304)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|63.88GB (54.09GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (5)</summary>

Extension|Author (truncated)|Version
---|---|---
admin-pack|Mic|0.0.2
agent|Mic|0.48.0
dacpac|Mic|1.8.0
import|Mic|1.3.1
profiler|Mic|0.11.0

</details>

**Comment by kburtram** [comment]
@scottkolle could you please confirm that "Preview Features" are enabled in the User Settings?  SQL Agent support is still in preview and will only display when that setting is true.

**Comment by scottkolle** [comment]
I searched the settings in the Azure Data Studio UI and the settings file show it is open too
    "workbench.enablePreviewFeatures": true,
    "datasource.connectionGroups": [
        {
The extension doesn’t’ show
***@***.***
Shows it is installed… tried rebooting, uninstalling/reinstalling the extensions and the Azure Data Studio

**Comment by scottkolle** [comment]
I tried to access the SQL Server Agent by going to Home on the server before I gathered these logs. Attached are the logs.
Thank you,
Scott

**Comment by Charles-Gagnon** [comment]
@scottkolle You need to attach the files through the Github UI - they don't support attachments through e-mail replies.

**Comment by scottkolle** [comment]
attached are the files
[consol.log](https://github.com/microsoft/azuredatastudio/files/7015921/consol.log)
[logs.zip](https://github.com/microsoft/azuredatastudio/files/7015922/logs.zip)

**Comment by Charles-Gagnon** [comment]
It looks like your connecting directly to a DB - Agent is currently configured to only show for connections to master (or default). If you make a new connection with those does it show up then?

**Comment by scottkolle** [comment]
Yes, that fixed it I see it now. I didn’t think about it needing to point to master. That resolves this.
Thank you,
Scott

**Comment by Charles-Gagnon** [comment]
@kburtram @abist Something to consider for improvement here - it would be great if the Agent stuff showed up even when connecting to a user DB as long as that user has access to the agent objects. SSMS appears to be doing this.

**Comment by scottkolle** [comment]
That was a little confusing, but makes since that it is on only when you are set to master, but yes a little confusing.
